### PR TITLE
OCPBUGS#13764 IPI bare metal supports by-path alias

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OCPBUGS-13764

Link to docs preview:
https://66965--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bare-metal-installation-disk-id

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@jadhaj @zaneb PTAL at this release note describing the new by-path alias for bare metal IPI installs. Do you think this enhancement justifies adding a release note for 4.14 GA? Thank you

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -132,6 +132,11 @@ You can now configure user-defined labels and tags in Google Cloud Platform (GCP
...
 
+[id="ocp-4-14-bare-metal-installation-disk-id"]
+==== Specifying installation disks using by-path device alias during bare metal installations
+
+You can now specify the installation disk using a by-path device alias, such as `deviceName: "/dev/disk/by-path/pci-0000:01:00.0-scsi-0:0:0:0"`, when you install a cluster on bare metal with installer-provisioned infrastructure. This type of disk alias persists across reboots. For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-the-install-config-file_ipi-install-installation-workflow[Configuring the install-config.yaml file].
```
Comment (zaneb) [inline release_notes/ocp-4-14-release-notes.adoc:138]: Note that this works on both IPI and the agent-based installer (different bugs, but both were added in 4.14 as well as having been backported). Link for the agent-based installer: https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#root-device-hints_preparing-to-install-with-agent-based-installer

**Review by zaneb — COMMENTED** [review]
This seems release note-worthy to me.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (bscott-rh) [reply release_notes/ocp-4-14-release-notes.adoc]: Thanks, I added mention of the Agent based installer

**Comment by mhanss** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -132,6 +132,11 @@ You can now configure user-defined labels and tags in Google Cloud Platform (GCP
 
 In {product-title} 4.14, you can install a cluster on Microsoft Azure in a restricted network for installer-provisioned infrastructure (IPI) and user-provisioned infrastructure (UPI). For IPI, you can create an internal mirror of the installation release content on an existing Azure Virtual Network (VNet). For UPI, you can install a cluster on Microsoft Azure by using infrastructure that you provide. For more information, see xref:../installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc#installing-restricted-networks-azure-installer-provisioned[Installing a cluster on Azure in a restricted network] and xref:../installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc#installing-restricted-networks-azure-user-provisioned[Installing a cluster on Azure in a restricted network with user-provisioned infrastructure].
 
+[id="ocp-4-14-bare-metal-installation-disk-id"]
+==== Specifying installation disks using by-path device alias
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:136]: Maybe one of these?
```suggestion
==== Specifying installation disks using by-path device aliases
```
or
```suggestion
==== Specifying installation disks using a by-path device alias
```

**Review by jeana-redhat — COMMENTED** [review]
One suggestion to think about

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (bscott-rh) [reply release_notes/ocp-4-14-release-notes.adoc]: Thanks, I went with "a by-path device alias".

**Pull request merged**